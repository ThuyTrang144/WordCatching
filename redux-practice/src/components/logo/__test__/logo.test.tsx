import { render } from "@testing-library/react";
import Logo from "..";

describe("Logo component", () => {
  test("should render Logo component", () => {
    const { getByTestId } = render(<Logo href="#" />);
    expect(getByTestId("logo")).toBeInTheDocument();
  });

  test("matches snapshot", () => {
    const { asFragment } = render(<Logo href="#" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
