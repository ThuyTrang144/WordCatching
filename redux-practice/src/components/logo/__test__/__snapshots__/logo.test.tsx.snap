// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="logo"
    data-testid="logo"
  >
    <a
      href="#"
    >
      <img
        alt="logo"
        class="logo__image"
        src=""
      />
    </a>
    <div
      class="logo__text-group"
    >
      <span
        class="logo__text"
      >
        Guess The Picture
      </span>
      <span
        class="logo__text--addon"
      >
        Gamespace
      </span>
    </div>
  </div>
</DocumentFragment>
`;
